### DISCUSSION TITLE
Use SSH key for RemoteFPing

### DISCUSSION THREAD

**Discussion by ActionParsnipNext** [oetiker/SmokePing#401] [Q&A]
Hi guys,
We have a need for RemoteFPing but would like to use SSH keys to connect to the initial host. Is there an option to give the private SSH key when establishing the connection per-host or is it one key to rule them all? If its a single key pair. Where should these reside please?
Thanks
-AndyW

**Comment by moetiker** [comment]
The probe is running on the smokeping server if you like to run probes on other server you can use smokping slaves ...

**Reply by ActionParsnipNext** [reply]
Yeah I get that but surely there is some form of authentication needed here, this is the question :)

How can we specify the SSH key to use, please?
